README.rst

```diff
@@ -14,20 +14,27 @@ uData customizations for Etalab / Data.gouv.fr.
 Compatibility
 =============
 
-udata-gouvfr requires Python 2.7+ and uData.
+**udata-gouvfr** requires Python 2.7+ and uData.
 
 
 Installation
 ============
 
-You can install udata-gouvfr with pip:
+Install [udata](https://github.com/opendatateam/udata).
 
-.. code-block:: console
+Remain in the same virtual environment (for Python) and use the same version of npm (for JS).
 
-    $ pip install udata-gouvfr
+Install **udata-gouvfr**:
 
-or with easy_install:
+    cd
```

Comment:
I think that line is unnecessary given that you probably don't want to install the project within your home folder (do you?)